File: ios/ObjectiveC/Beacon/3-Alert/Starter/beaconAlertStarter/ViewController.m
//
//  ViewController.m
//  beaconAlertStart
//
//  Created by sarra srairi on 29/03/2016.
//  Copyright © 2016 sarra srairi. All rights reserved.
//

#import "ViewController.h"
#import "AlertViewControllerAction.h"

@interface ViewController () {
 
    ATBeaconContent *currentAlertBeaconContent;
}

@end

@implementation ViewController

-(void)viewDidLoad {
    [super viewDidLoad];
 
    // Do any additional setup after loading the view, typically from a nib.
}

-(void)viewDidAppear:(BOOL)animated{
    [super viewDidAppear:animated];
}
-(void)viewDidDisappear:(BOOL)animated{
    [super viewDidDisappear:animated];
}

-(void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

-(void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender{
    if ([segue.identifier isEqualToString:@"alert"]) {
        AlertViewControllerAction *controllerAlert= (AlertViewControllerAction *)[segue destinationViewController];
        controllerAlert.alertBeaconContent = currentAlertBeaconContent;
    }
}

@end

File: ios/SWIFT/Beacon/9-Alert-Strategy/Starter/AlertStrategy/ViewController.swift
//
//  ViewController.swift
//  AlertStrategy
//
//  Created by sarra srairi on 11/08/2016.
//  Copyright © 2016 R&D connecthings. All rights reserved.
//

import UIKit
import ATAnalytics
import ATLocationBeacon

class ViewController: UIViewController {
    
    @IBOutlet weak var txtAlertMessage: UILabel!
 
    @IBOutlet weak var actionTxt: UILabel!
    var currentAlertBeaconContent: ATBeaconContent!
  
    override func viewDidLoad() {
        super.viewDidLoad()
        
        let alertTimeFilter: FirstWayAlertTimerStrategy = FirstWayAlertTimerStrategy(maxTime: 60 * 1, DelayAfterDetectingBeaconToCreateAlert: 30000)
        

        
        // Do any additional setup after loading the view, typically from a nib.
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    

}

